frontend/src/pages/exploreApplication/ExploreApplications.tsx

```diff
@@ -58,7 +52,7 @@ const ExploreApplicationsInner: React.FC<ExploreApplicationsInnerProps> = React.
             />
           }
         >
-          <DrawerContentBody className={bodyClasses}>
+          <DrawerContentBody style={{ display: 'flex', flexDirection: 'column' }}>
```

Comment:
A little concerning we are doing this -- can we not get a proper layout in place to handle flex? Why are we directly mutating a PF component to do this?

Reply:
I tried to apply the `Flex` layout to the DrawerContentBody, but it doesn't work out well, it will cause the page scrollable even if there are only a few cards on the page. I am trying to find out the reason, but I guess it's because the `section` element is not the child of DrawerContentBody anymore, which causes it inherits some different height that causes the expansion of the page height.

Reply:
🤔 Hmmm -- well try to do away with the flex. I could help take a look if you're out of ideas. We shouldn't really re-associate PF components with a different display type.

Reply:
So if I go without any custom inline styles, it will be like: 
<img width="1474" alt="Screenshot 2023-01-09 at 3 58 13 PM" src="https://user-images.githubusercontent.com/37624318/211407076-8cded4eb-4abe-4267-b4e9-c9836ee99e83.png">
The content page section cannot fill the page up because the DrawerContentBody is not flex layout.

If I go with the current change, which is the custom inline styles, it will be like:
<img width="1474" alt="Screenshot 2023-01-09 at 3 59 51 PM" src="https://user-images.githubusercontent.com/37624318/211407351-babdb903-a473-4dea-a58d-88f35e92c3e8.png">
This will fill the page height up and not scrollable, which is good.

If I use the PF Flex layout instead, the content becomes scrollable, which I cannot find out the reason:
<img width="1474" alt="Screenshot 2023-01-09 at 4 02 21 PM" src="https://user-images.githubusercontent.com/37624318/211407726-ae950db4-0a7a-4b60-a79b-7710999ca8c6.png">

I will appreciate that if you could help me take a look at it! @andrewballantyne

Reply:
Thanks for the explanation @DaoDaoNoCode I'll take a look tomorrow.

Reply:
Okay, missed taking a look at this yesterday -- I think the issue boils down to the lack of `Page` when using `PageSection`s and you're reacting to that by mimicking the part of `Page` the `PageSection` needs -- which is Flex.

We should talk about solutions tomorrow.